docs/sources/datasources/tempo/_index.md

```diff
@@ -57,51 +57,52 @@ The **Trace to logs** setting configures the [trace to logs feature]({{< relref
+| ---------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| **\_\_tags**           | This variable is special because it uses the tag mapping in from the UI to create a label matcher string in the specific data source syntax. It uses only tags that are present in the span so the link will still be created even if only one of those tags is present in the span. You can use this if not all the tags are required for the query to be useful. |
+| **\_\_span.spanId**    | The ID of the span.                                                                                                                                                                                                                                                                                                                                                |
+| **\_\_span.traceId**   | The ID of the trace.                                                                                                                                                                                                                                                                                                                                               |
+| **\_\_span.duration**  | The duration of the span.                                                                                                                                                                                                                                                                                                                                          |
+| **\_\_span.name**      | Name of the span.                                                                                                                                                                                                                                                                                                                                                  |
+| **\_\_span.tags**      | Namespace for the tags in the span. To access a specific tag named "version" you would use `${__span.tags.version}`. In case the tag contains dot you have to access it as `${__span.tags["http.status"]}`.                                                                                                                                                        |
```

Comment:
```suggestion
| **\_\_span.tags**      | Namespace for the tags in the span. To access a specific tag named `version`, you would use `${__span.tags.version}`. In case the tag contains dot, you have to access it as `${__span.tags["http.status"]}`.                                                                                                                                                        |
```